src/test/java/org/aksw/simba/squirrel/analyzer/impl/AnalyzerTest.java

```diff
@@ -0,0 +1,71 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+import java.io.File;
+import java.net.InetAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+import java.util.Iterator;
+
+import org.aksw.simba.squirrel.analyzer.Analyzer;
+import org.aksw.simba.squirrel.collect.SimpleUriCollector;
+import org.aksw.simba.squirrel.collect.UriCollector;
+import org.aksw.simba.squirrel.data.uri.CrawleableUri;
+import org.aksw.simba.squirrel.data.uri.UriType;
+import org.aksw.simba.squirrel.data.uri.serialize.java.GzipJavaUriSerializer;
+import org.aksw.simba.squirrel.sink.Sink;
+import org.aksw.simba.squirrel.sink.impl.mem.InMemorySink;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+@SuppressWarnings("deprecation")
+public class AnalyzerTest {
```

Comment:
I am not sure why this test class is in this branch... it has nothing to do with the feature you wanted to implement, right? :thinking: